app/models/oral_history_content/ohms_xml.rb

```diff
@@ -94,36 +94,90 @@ def initialize(xml_node)
 
     private
 
-    # A hash where key is an OHMS line number. Value is a Hash containing
-    # :word_number and :seconds .
-    #
-    # We parse the somewhat mystical OHMS <sync> element to get it.
+    # A hash where key is an OHMS line number. Value is a hash containing
+    # word_number and seconds.
+    def parse_sync!
+      remove_consecutive_timecodes(raw_timecodes)
+    end
```

Comment:
Not a big deal, but any reason to have this just be a method that calls another method, instead of only one method? But there might be a reason!

Reply:
Good call.

Reply:
I mean there might actually be a reason, it might be fine. Maybe I was just reaching for something to comment on I wouldn't worry about it actually.